analyses/cell-type-wilms-tumor-14/scripts/utils/01_anchor_transfer_seurat_functions.R

```diff
@@ -106,4 +107,10 @@ plot_anchorTrans <- function(path_anal, scratch_out_dir, results_out_dir, plots_
                    filename = filename,
                    width = ifelse(length( pred_ids ) < 15,9,12), 
                    height = ifelse(length( pred_ids ) < 15,6,9))
+  
+  # save pngs
+  ggsave(p, file = filename_core,
+         device = "png", dpi = 72,
```

Comment:
72 dpi is pretty low these days, even for just on screen viewing. We could stay with the default 300dpi, but 200 is probably a good compromise to shrink the files a bit.
```suggestion
         device = "png", dpi = 200,
```